src/scripts/main.js

```diff
@@ -354,7 +354,36 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const getTable = document.querySelector('.dashboard');
 
-// write your code here
+// If the table exists, proceed to populate it
+if (getTable) {
+  // Loop through each person in the 'people' array
+  people.forEach((person) => {
+    // Create a new table row
+    const newRow = document.createElement('tr');
+
+    // Create table cells and append them to the row
```

Comment:
Remove comments. Try to write obvious code that explains all we need without comments)